src/Tokenizer/Analyzer/BlocksAnalyzer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tokenizer\Analyzer;
+
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Kuba Werłos <werlos@gmail.com>
+ *
+ * @internal
+ */
+final class BlocksAnalyzer
+{
+    /**
+     * @param Tokens   $tokens
+     * @param null|int $openIndex
+     * @param null|int $closeIndex
+     *
+     * @return bool
+     */
+    public function isBlock(Tokens $tokens, $openIndex, $closeIndex)
+    {
+        if (!$tokens->offsetExists($openIndex)) {
```

Comment:
https://www.php.net/manual/en/splfixedarray.offsetexists.php
it expects `int` as parameter, you are giving `null|int`.

Reply:
Yet, it there are tests for `null` and are passing.

Anyway, I've added the check for `null` to make it more clear.